### PR TITLE
TFAC-1117 Presentational Components for new SFAC Tab Page Component

### PR DESCRIPTION
**Pull request by jedtan**

### CODE REVIEW

File: src/components/SfacActivityNotification.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  totalSearches: 100,
+  searchesToday: 5,
+}
+
+export const newStatus = Template.bind({})
+newStatus.args = {
+  activityState: 'newStatus',
+  totalSearches: 100,
+  searchesToday: 5,
+}
```
Comment (kmjennison) [inline src/components/SfacActivityNotification.stories.jsx:29]: Please also provide a set with a different cause name, e.g. "Reproductive Health"

**Review by kmjennison — CHANGES_REQUESTED** [review]
Good progress on this! Requested some changes in UI and props. Also see Chromatic comments for UI feedback.

File: src/components/SfacActivityNotification.js [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+  searchCount: {
+    fontWeight: 900,
+  },
+}))
+const SfacExtensionSellNotification = ({
+  activityState,
+  searchesToday,
+  totalSearches,
+}) => {
+  const [open, setOpen] = useState(true)
```
[the remark this answers is not in the document]
Comment (jedtan) [reply src/components/SfacActivityNotification.js:74]: I actually have this refactor in the next PR. Going to be lazy and keep it in that separate branch.

File: src/components/SfacActivityNotification.js [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+  if (activityState === 'inactive') {
+    buttons = (
+      <div className={classes.buttonsWrapper}>
+        <Link
+          to={SFAC_FEEDBACK_LINK}
+          className={classes.noLink}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          FEEDBACK
```
Comment (kmjennison) [inline src/components/SfacActivityNotification.js:109]: Thanks for changing to links! However, these should also still use the Button component (consistent UI when interacting, likely better accessibility).

File: src/components/SfacActivityNotification.stories.jsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+const useStyles = makeStyles((theme) => ({
+  widthDiv: {
+    width: theme.spacing(50),
+  },
+}))
+
+const Template = (args) => {
+  const classes = useStyles()
+  return <SfacActivityNotification {...args} className={classes.widthDiv} />
```
Comment (kmjennison) [inline src/components/SfacActivityNotification.stories.jsx:18]: Perfect, thanks for this edit!

**Review by kmjennison — DISMISSED** [review]
UI is looking great! Please just keep using the Button components within the links, then this should be good to ship. Thanks!

**Pull request merged**